src/components/Contacts/ContactPartnershipStatus/ContactLateStatusLabel/ContactLateStatusLabel.test.tsx

```diff
@@ -1,54 +1,121 @@
 import React from 'react';
 import { ThemeProvider } from '@mui/material/styles';
 import { render } from '@testing-library/react';
+import { Settings } from 'luxon';
 import theme from '../../../../theme';
-import {
-  ContactLateStatusEnum,
-  ContactLateStatusLabel,
-} from './ContactLateStatusLabel';
+import { ContactLateStatusLabel } from './ContactLateStatusLabel';
+
+Settings.now = () => new Date(2020, 1, 1).valueOf();
 
 describe('ContactLateStatusLabel', () => {
-  it('should render on time', () => {
+  it('should render on time when both dates are in the future', () => {
     const { getByText } = render(
       <ThemeProvider theme={theme}>
-        <ContactLateStatusLabel lateStatusEnum={ContactLateStatusEnum.OnTime} />
+        <ContactLateStatusLabel
+          lateAt="2020-02-06T12:00:00Z"
+          pledgeStartDate="2020-02-04T12:00:00Z"
```

Comment:
The GraphQL type of these fields is `ISO8601Date`, which means that on the server, the dates won't have the time or timezone fields. These strings would match the `ISO8601DateTime` type. Do you mind removing the time information from these and all the others in this file?

```suggestion
          lateAt="2020-02-06"
          pledgeStartDate="2020-02-04"
```